cmd/app.go

```diff
@@ -637,6 +610,67 @@ func resolveTemplatePath(templateName string) bool {
 	return templateName == "." || strings.Contains(templateName, "\\\\") || strings.Contains(templateName, "/")
 }
 
+func walkTemplateDirectory(templatePath string) (chartName string, files map[string]string, err error) {
```

Comment:
This function should not do naked returns, it is doing too much to easily grok what is happening with 2 nested anonymous functions.

Reply:
Refactored